wenet/bin/train.py

```diff
@@ -77,16 +77,25 @@
 
     distributed = args.world_size > 1
 
-    # Init dataset and data loader
-    collate_func = CollateFunc(**configs['collate_conf'],
-                               **configs['spec_aug_conf'],
-                               cmvn=args.cmvn)
-    cv_collate_conf = copy.copy(configs['collate_conf'])
+    raw_wav = configs['raw_wav']
+
+    train_collate_func = CollateFunc(**configs['collate_conf'],
+                                     raw_wav=raw_wav,
+                                     cmvn=args.cmvn)
+
+    cv_collate_conf = copy.deepcopy(configs['collate_conf'])
+    # no augmenation on cv set
     cv_collate_conf['spec_aug'] = False
-    cv_collate_func = CollateFunc(**cv_collate_conf, cmvn=args.cmvn)
+    cv_collate_conf['feature_dither'] = 0.0
+    cv_collate_conf['speed_perturb'] = False
+    cv_collate_conf['wav_distortion_conf']['wav_distortion_rate'] = 0
```

Comment:
read it from train.yaml

Reply:
I follow previous logics that use the same configs for the train/cv dataset, and disable the augmentation related config for the CV dataset.  Do you mean using separated configs for train/cv/test dataset?